great_expectations/rule_based_profiler/config/base.py

```diff
@@ -0,0 +1,282 @@
+import copy
```

Comment:
@cdkini I would prefer -- at least initially -- to put this into our overall `data_context/types/base.py` along with the other `Schema` constructs (unless this is a step in the effort to break that large file up).  Thanks

Reply:
@cdkini I am not sure -- on one hand, I like to see all schemas in one place; on the other hand, the main `base.py` is very large.  The decision here has to adhere to our future strategy -- so please decide together with @donaldheppner and let everybody know.  Thanks!

Reply:
@cdkini One opinion: If you end up getting into an `import` tail chasing by having a separate file, then put it into the main one (because you have to import some top-level classes from the main `base.py`).  Thanks!

Reply:
I think the aligned strategy is to keep it here but will revisit if it becomes an issue!